tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -546,6 +561,10 @@ module.exports = {
     },
     sidebarPrivateLinkIconCopied: {
       selector: '#files-sidebar-private-link-icon-copied'
+    },
+    sidebarSharingContent: {
+      selector: '//div[@class="uk-active"]',
```

Comment:
This xpath is not clear. Please use a better xpath to select the sharing sidebar content.
This basically checks for active tabs in the sidebar. If there are other tabs other than sharing ones this will still find that one.